src/app/personside/infotabs/meldinger/traadvisning/verktoylinje/oppgave/OppgaveSkjemaElementer.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { ChangeEvent } from 'react';
+import Select from 'nav-frontend-skjema/lib/select';
+import RadioPanelGruppe from 'nav-frontend-skjema/lib/radio-panel-gruppe';
+import {
+    GsakTema,
+    GsakTemaOppgavetype,
+    GsakTemaUnderkategori,
+    OppgavePrioritet
+} from '../../../../../../../models/meldinger/oppgave';
+import Textarea from 'nav-frontend-skjema/lib/textarea';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import { LenkeKnapp } from '../../../../../../../components/common-styled-components';
+import { OppgaveProps, OppgaveSkjema } from './oppgaveInterfaces';
+import styled from 'styled-components';
+
+const KnappStyle = styled.div`
+    display: flex;
+    justify-content: space-between;
+`;
+
+const SkjemaStyle = styled.div`
+    .inputPanelGruppe__inner {
+        display: flex;
+        > * {
+            flex-grow: 1;
+        }
+    }
+`;
+
+export function OppgaveSkjemaElementer(props: OppgaveProps & { form: OppgaveSkjema }) {
+    const valgtTema = props.form.state.valgtTema;
+    return (
+        <SkjemaStyle>
+            <Select
+                label={'Tema'}
+                onChange={event => props.form.actions.settValgtTema(hentValgtTema(props.gsakTema, event))}
+            >
+                {lagTemaOptions(props.gsakTema)}
+            </Select>
+            <Select
+                label={'Gjelder'}
+                onChange={event => props.form.actions.settValgtUnderkategori(hentValgtUnderkategori(event, valgtTema))}
+            >
+                {lagUnderkategoriOptions(valgtTema)}
+            </Select>
+            <Select
+                label={'Type oppgave'}
+                onChange={event => props.form.actions.settValgtOppgavetype(hentValgtOppgavetype(event, valgtTema))}
+            >
+                {lagOppgavetypeOptions(valgtTema)}
+            </Select>
+            <RadioPanelGruppe
+                radios={[
+                    { label: 'Høy', value: OppgavePrioritet.HOY, disabled: !valgtTema },
```

Comment:
Trenger du å disable disse?

Reply:
Kanskje. Det er for å signalisere at du må velge et sakstema før du får lov til å gjøre noe annet.

Reply:
Mm, disabling er i utgangspunktet en uting og bør unngås eller løses på andre måter. Prioritet er vel uansett ikke avhengig av andre valg, så kan vel likegreit la den være tilgjengelig..?